manifests/j/JanDeDobbeleer/OhMyPosh/3.166.1/JanDeDobbeleer.OhMyPosh.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.166.1
+InstallModes:
+- silent
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```
You didn't do what ryfu-msft suggested https://github.com/microsoft/winget-pkgs/pull/18482#issuecomment-866415557

Reply:
I know, I forgot to adjust that. These PR's are automated from my codebase. The weird thing remains that this did work before and no changes were made to the definition, nor the Inno installer. So, it's not really an explanation.